index.html

```diff
@@ -128,89 +128,9 @@ <h3>CC Ignite</h3>
-            </div>
-
-
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/jose-mourinho-1.jpeg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>José Mourinho</h3>
-                    <h4>Coach</h5>
-                        <hr>
-                        <p>59 years old <br>Once dubbed "The Special One" by the British media,
-                            Mourinho is widely considered to be among the greatest.</p>
-                </div>
-            </div>
-
-
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/pep-guardiola.jpeg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>Pep Guardiola</h3>
-                    <h4>Assistant Coach</h5>
-                        <hr>
-                        <p>51 years old <br>Spanish professional football manager and former player, who is the current
-                            manager of Premier League club Manchester City.</p>
-                </div>
-            </div>
 
+            <!-- Add dynamicly the speakers in the Js file -->
 
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/Zinedine_Zidane.jpeg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>Zinedine Zidane</h3>
-                    <h4>Assistant Coach</h5>
-                        <hr>
-                        <p>50 years old <br>French professional football manager and former player who played as an
-                            attacking midfielder.</p>
-                </div>
-            </div>
-
-
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/gregorgy-dupont.jpeg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>Grégory Dupont</h3>
-                    <h4>Fitness Coach</h5>
-                        <hr>
-                        <p>50 years old <br>Sports physical trainer, he is known as the scientist since he usually plans
-                            his
-                            workouts based on scientific research.</p>
-                </div>
-            </div>
-
-
-            <div class="staff-card">
-                <div class="staf-img">
-                    <img src="img/antonio-pintus.jpg" alt="Academy_Staff">
-                </div>
-                <div class="staff-intro">
-                    <h3>Anthony Pintus</h3>
-                    <h4>Athletic Coach</h5>
-                        <hr>
-                        <p>59 years old <br>Italien physical trainer, known as "the sergeant" for the harshness of his
-                            methods,</p>
-                </div>
-            </div>
         </div>
```

Comment:
- [ ] In your home page for mobile version I can see the footers but in the given design there is no footer for the mobile version so kindly remove both partners and last footer from your home page just in the mobile version. ✅✨🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/210465277-aaf31c17-7b1d-48c7-959c-c57624a5a9c1.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/210465302-00c881df-11b9-4482-9ed4-7a3711343869.png)